.github/labeler.yml

```diff
@@ -10,6 +10,13 @@ CI/run-network-audit:
 - chromium_src/**/*features.cc
 - patches/chrome-VERSION.patch
 
+CI/storybook-url:
```

Comment:
`CI/run-storybook`

Reply:
ditto, I think still always running storybook is ok